src/wirecloud/platform/iwidget/utils.py

```diff
@@ -104,10 +105,10 @@ def update_screen_size_value(model: WidgetPositionsConfig, data: LayoutConfig, f
     value = getattr(data, field)
 
     if type(value) is not int:
-        raise TypeError(f'Field {field} must contain a number value')  # TODO translate this
+        raise TypeError(_(f'Field {field} must contain a number value'))
```

Comment:
No puede haber traducciones con reemplazo antes de traducir, debe ser:
```suggestion
        raise TypeError(_(f'Field %(field)s must contain a number value') % {"field": field})
```